algorithms/spot_finding/factory.py

```diff
@@ -464,6 +464,7 @@ def from_parameters(params=None, experiments=None):
             max_spot_size=params.spotfinder.filter.max_spot_size,
             no_shoeboxes_2d=no_shoeboxes_2d,
             min_chunksize=params.spotfinder.mp.min_chunksize,
+            is_stills=is_stills,
```

Comment:
Without further comment I note that this is parameter 19 to a constructor 🙄

To me this suggests that this code needs refactoring (as well as emphasising that the parameter addition is probably not a good thing in the wider scope)

I am not in any sense asking for this to be addressed here and now, though I think raising an issue following this code being merged would be appropriate and justified.

Reply:
Issue raised